lib/offsite_payments/integrations/quickpay.rb

```diff
@@ -217,7 +217,7 @@ def parse(post)
             post.split(/-{20,40}\w{6,24}[\n\r]*/m).each do |part|
               part.scan(/([^\n\r]+)[\n\r]+([^\n\r]*)/m) do |header, value|
                 if header.match(/name=["'](.*)["']/)
-                  params[$1] = value.strip
```

Comment:
Can you explain this change?

Reply:
Commit says "Do not strip notification post data.".

Commit was over 3 years ago, by someone not with our organization any more. So the reasoning is lost to time at this point.

Reply:
So do you still want to keep it? I mean I don't see the value of changing this legacy integration since "it works" and you will rely more on the v10 in the future.

Reply:
Change undone.